test/paper-dialog-behavior.html

```diff
@@ -359,8 +354,6 @@ <h2 id="header">Dialog</h2>
               // causing a "Maximum call stack size exceeded" error.
               // Wait 50ms and verify this does not happen.
               Polymer.Base.async(function() {
-                // Should not enter in an infinite loop.
-                assert.equal(spy.callCount, 1, 'focused only once');
```

Comment:
Well, then at least leave the `// Should not enter in an infinite loop.` comment in. this test isn't doing asserts otherwise so that's a bit weird